### PR TITLE
Added other possible ways to use different versions of delfin image 

### PR DESCRIPTION
**Pull request by vineela1999**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind new feature
> /kind bug fix
> /kind cleanup
> /kind revert change
> /kind design
> /kind documentation

kind enhancement

**What this PR does / why we need it**:
This PR includes other ways of getting image into node and using the specific version of image in object files
And config map yaml file configurations 
**Which issue(s) this PR fixes**:

Fixes #

**Test Report Added?**:
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
> /kind TESTED
> /kind NOT-TESTED

**Test Report**:

**Special notes for your reviewer**:

### CODE REVIEW

File: content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md [hunk 1]
```diff
@@ -91,18 +91,41 @@ Server Version: v1.21.1 <br />
 
 ---
 
-### Build the delfin docker image locally
+### Regarding the delfin image of deployment
```
Comment (sushanthakumar) [inline content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md:94]: I think u can change the sequence like below
1) Add the section : Get all kubernetes object files required for delfin deployment:
2) After the initial clone part , introduce this "delfin image of deployment"  with all options (default, change version tag, build from source)
3) Then continue from "Commands to bring up the delfin services"

Comment (vineela1999) [reply content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md]: changed the order

File: content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md [hunk 2]
```diff
@@ -91,18 +91,35 @@ Server Version: v1.21.1 <br />
...
+**To use a specific version of delfin replace the tags of delfin image in `delfin-api-deployment.yaml` ,`delfin-task-deployment.yaml`, `delfin-exporter-deployment.yaml` and `delfin-alert-deployment.yaml` files to desired tag(optional)** <br />
+
+```bash
+# replace tag with required tag in each of the above mentioned files
+image: sodafoundation/delfin:<tag required>
+```
+{{% /notice %}}
+
+
+#### If the user wants to build image from source code manually(optional)
```
Comment (anvithks) [inline content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md:110]: nit: Please rephrase to  `Build image from source code manually (optional)`

File: content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md [hunk 3]
```diff
@@ -91,18 +91,35 @@ Server Version: v1.21.1 <br />
...
 To build the delfin docker image, source code and  Dockerfile are needed. Dockerfile contains instructions on how the image is built.<br />
 **Download the source code of delfin:**
 ```bash
 git clone https://github.com/sodafoundation/delfin.git
 ```
-
-**Build Delfin Image with k8s tag locally:**
+**Build Delfin Image and get into local:**
```
Comment (anvithks) [inline content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md:116]: nit: Image -> image
`and get into local` -> `and get into local environment`

File: content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md [hunk 4]
```diff
@@ -112,28 +129,28 @@ docker images
...
 
-### Get all kubernetes object files required for delfin deployment:
-```bash
-#Navigate out of delfin folder
-cd ..
-git clone https://github.com/sodafoundation/examples.git
-# examples/delfin-kubernetes/deploy directory contains all the object files of delfin k8s delpoyment
-```
 ### Commands to bring up the delfin services:
 All the object files gets added to the kubernetes cluster<br />
```
Comment (anvithks) [inline content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md:142]: gets added -> get added

File: content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md [hunk 5]
```diff
@@ -112,28 +129,28 @@ docker images
...
 
+#### Create all config Maps
```
Comment (anvithks) [inline content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md:144]: config Maps -> configMaps

File: content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md [hunk 6]
```diff
@@ -156,8 +173,12 @@ kubectl get all
 ---
 
 ### To monitor the performance metrics on prometheus
-Follow the below site:
-
+Open the `configMap.yaml` file 
+```bash
+# Uncomment the line in configMap.yaml file
+performance_exporters = PerformanceExporterPrometheus, PerformanceExporterKafka
```
Comment (anvithks) [inline content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md:179]: Are both exporters needed or only the `PerformanceExporterPrometheus` is needed?
During SODA installation with delfin we enable the exporter that is required but not both.

**Review by anvithks — CHANGES_REQUESTED** [review]
Please check the individual comments. Overall there are a few typos.

File: content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md (hunk 4 above)
Comment (vineela1999) [reply content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md]: changed

File: content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md (hunk 3 above)
Comment (vineela1999) [reply content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md]: changed

File: content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md (hunk 2 above)
Comment (vineela1999) [reply content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md]: changed

File: content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md [hunk 7]
```diff
@@ -1,5 +1,5 @@
 ---
-title: delfin deployment on kubernetes
+title: Deployment on kubernetes cluster
 description: "Dedploying delfin services on kubernetes ."
```
[the remark this answers is not in the document]
Comment (vineela1999) [reply content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md]: changed

File: content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md (hunk 5 above)
Comment (vineela1999) [reply content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md]: changed

File: content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md (hunk 6 above)
Comment (vineela1999) [reply content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md]: changed

**Review by joseph-v — APPROVED** [review]
LGTM

**Review by anvithks — APPROVED** [review]
LGTM

**Pull request merged**